api/v1alpha1/istio_types.go

```diff
@@ -200,7 +200,13 @@ const (
 // +kubebuilder:printcolumn:name="Version",type="string",JSONPath=".spec.version",description="The version of the control plane installation."
 // +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp",description="The age of the object"
 
-// Istio represents an Istio Service Mesh deployment
+// Istio represents an Istio Service Mesh deployment consisting of one or more
```

Comment:
@luksa just a thought...when we say "a cluster-admin creates" should we say "a user account with the cluster-admin role"

Reply:
Hm, my intent was not to specifically say that a cluster-admin role is needed, only that a cluster administrator creates the object. I started just with "user", but then realized I could make it more specific, but it's not completely true that only users with the cluster-admin role can create the object. Anyone that has the `create Istio` permission can do it (but initially only the cluster-admin has this permission). 

Anyway, the point of the description is to explain what the user does vs what the operator does, so maybe I should say "user" instead of "cluster-admin"?

Reply:
Ah, ok, thanks for the clarification. I'm fine if you want to say user instead of cluster-admin, and, in general, I like the distinction of what the user does vs what the operator does.